components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -221,4 +258,119 @@ Should you have a need to revert back to IAM v1, it is quick and easy to do:
 $ chef-automate iam reset-to-v1
 ```
 
-Note that this discards your IAM v2 policies and roles, and re-configures Chef Automate to use your v1 policies again.
+Note that this discards your IAM v2 policies, roles, and projects, and re-configures Chef Automate to use your v1 policies again.
+
+<!-- TODO move this next to the rest of the projects walkthrough -->
+## Assigning Projects to Ingested Resources
+
+While projects can be directly assigned to resources created within Automate, ingested resources, including Compliance reports, events, and nodes, are created outside of Automate and therefore must be tagged with projects a different way.
```

Comment:
lets please avoid _tag_ or _tagged_ language with projects, instead lets try and use language like _moving something into a project_, or _associating something with a project_